.github/workflows/goreleaser-ci.yml

```diff
@@ -0,0 +1,35 @@
+
+name: GoReleaser CI
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - '**'
+  workflow_dispatch:
+
+jobs:
+  goreleaser_ci_build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.17.1]
```

Comment:
Let's use 1.17.0 here, so that we're using the same version everywhere. (Unelss we've updated to 1.17.1 in prod and I didn't know about it, in which case we should update the go version used in the rest of our CI.)

Reply:
Changed to 1.17.0